ocaml/ocamltest/ocaml_actions.ml

```diff
@@ -794,19 +794,22 @@ let cc =
 
 let run_expect_once input_file principal log env =
   let expect_flags = Sys.safe_getenv "EXPECT_FLAGS" in
-  let repo_root = "-repo-root " ^ Ocaml_directories.srcdir in
+  let stdlib_dir = Ocaml_directories.srcdir ^ "/stdlib" in
   let principal_flag = if principal then "-principal" else "" in
   let commandline =
   [
     Ocaml_commands.ocamlrun_expect_test;
     expect_flags;
     flags env;
-    repo_root;
     principal_flag;
     input_file
   ] in
+  let environment = Array.concat [
+    default_ocaml_env;
+    [| (Printf.sprintf "OCAMLLIB=%s" stdlib_dir) |]
```

Comment:
What about this PR makes this necessary?

Reply:
Otherwise, `#directory +stable` does not work in expect tests, and we can't load the `Stable` module. But I'm now looking into just fixing `include stable` for expect tests.